### ISSUE TITLE
Confusing documentation for query parameters

### ISSUE THREAD

**Issue by IlyaHalsky** [heartexlabs/label-studio#4875]
**Describe the bug**
Query parameters under `https://labelstud.io/api#tag/Export/operation/api_projects_export_read` are listed in snake case, but in examples they are in camel case

**Screenshots**
[Example](https://prnt.sc/mOjwIyMWmULH)

**Additional context**
Maybe I'm missing something...

**Comment by hogepodge** [comment]
Indeed, this is confusing. I'm taking it to our docs and engineering team to get a better understanding of what's going on.

**Comment by hogepodge** [comment]
We've identified this as a priority to fix, and will be rolling out consistent variable naming (with existing backward compatibility) soon.
